docs/python/tutorial-azure-functions.md

```diff
@@ -381,6 +383,89 @@ After your first deployment, you can make changes to your code, such as adding a
 
 1. Once deployment finishes (it takes a few minutes!), the **Output** window shows the public endpoints with which you can repeat your tests.
 
+## Add a binding to write messages to Azure storage
+
+A "binding" is how you connect an Azure function to other Azure resources, such as storage. A binding is defined in the `function.json` file and represents both input and output. A function can use multiple input and output bindings. (To learn more, see [Azure Functions triggers and bindings concepts](https://docs.microsoft.com/azure/azure-functions/functions-triggers-bindings) in the Azure documentation.)
+
+In this section, you add a storage binding to the HttpExample function created earlier in this tutorial. The function uses this binding to write messages to storage with each request.
+
+1. Sync the remote settings for your Azure Functions project into your `local.settings.json` file: in the **Azure Explorer** in VS Code, right-click the **Application Settings** node and select the **Download remote settings...** command. Open `local.settings.json` and check that it contains a value for `AzureWebJobsStorage`. That value is the connection string for the storage account.
+
+1. In the `HttpExample` folder, right-click the `function.json`, select **Add binding**, and select or provide the following values for the prompts:
```

Comment:
[Nice to have] What do you think about providing a screenshot of the right click on "function.json" and **Add binding** action? Ad tagged, this is a nice to have and can punt for later if that's better.

Reply:
Good idea, will add that.